plugins/rapture/nexus-rapture-plugin/src/main/resources/static/rapture/NX/controller/SettingsForm.js

```diff
@@ -95,6 +103,22 @@ Ext.define('NX.controller.SettingsForm', {
     }
   },
 
+  /*
+   * @private
+   * Reset the dirty state of the form
+   */
+  resetDirtyState: function (form) {
```

Comment:
Thanks @kellyrob99. I’ll look into this. Hopefully it’ll fix a few forms that still show as dirty even though they aren’t (Repository details, Route details).